nailgun/entities.py

```diff
@@ -4005,9 +4006,12 @@ def update(self, fields=None):
 
     def update_payload(self, fields=None):
         """Wrap submitted data within an extra dict."""
-        return {
-            u'organization': super(Organization, self).update_payload(fields)
-        }
+        org_payload = super(Organization, self).update_payload(fields)
+        payload = {u'organization': org_payload}
+        if 'redhat_repository_url' in org_payload:
+            rh_repo_url = org_payload['redhat_repository_url']
```

Comment:
If I move instead of copy this value, it would affect others who may have been using it. Is there little risk of that? If so, I will move it.